andela/travel_tool_front
# #160370625 Write test for edit comment functionality

#### What does this PR do?
This PR adds the test to cater for edit comment feature.

#### Description of Task to be completed?
Write test for edit comment feature, the following test files contain the written test;      

1.        src/components/RequestsModal/CommentBox/__tests__/CommentBox.test.js
2.         src/components/RequestsModal/UserComments/__tests__/UserComments.test.js
3.         src/redux/actionCreator/__tests__/commentsActions.test.js
4.         src/redux/middleware/__tests__/commentsSaga.test.js
5.         src/redux/reducers/__tests__/comments.test.js
6.         src/services/__tests__/CommentsAPI.test.js 

#### How should this be manually tested?
- Clone the repository
- Switch to the repo directory - `cd travel_tool_front`
- Run `make test` to run all the test files
- On the console, you should be able to see the test coverage displaying test covered 
#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#160370625](https://www.pivotaltracker.com/story/show/160370625)

#### Screenshots (if appropriate)
None

#### Questions:
None
